File: Week-03/1. Primitive/Solutions/ROHIT KUMAR(GCS1940022)/solution04.py
#code
t=int(input())
for i in range(t):
    n,p=map(int,input().split())
    arr=list(map(int,input().split()))
    count=0
    for j in range(n):
        for k in range(j+1,n):
            if arr[j]+arr[k]==p:
                count+=1
    print(count)

File: Week-03/1. Primitive/Solutions/BhaveshSoni (GCS-1840039)/solution 04.cpp
#include<iostream>
using namespace std;
int main()
 {
    int T;
    cin>>T;
    for(int i=0;i<T;i++)
    {
       int n,k;
       cin>>n>>k;
       int arr[n];
       for(int j=0;j<n;j++)
       cin>>arr[j];
       int count=0;
       for(int j=0;j<n;j++)
           {
               int a1=arr[j];
               int l=0;
               while(l<n&&l!=j)
                 {
                    int a2=arr[l];
                    if((a1+a2)==k)
                    count++;
                    l++;
                 }
           }
        cout<<count<<endl;
    }
    //code
	return 0;
}
